docker/openemr/flex-3.10/autoconfig.sh

```diff
@@ -167,7 +167,15 @@ if [ -f /etc/docker-leader ] ||
 
         # install php dependencies
         composer install
+        if [ "$DEVELOPER_TOOLS" == "yes" ]; then
+            composer global require "squizlabs/php_codesniffer=3.*" 
+            composer require "phpunit/phpunit=8.*"
+            composer require "symfony/panther=^0.6" 
 
+             # optimize
+             composer dump-autoload -o
```

Comment:
line 200 already does this, so can drop above 2 lines

Reply:
Done.